packages/transitive-overlay/src/index.tsx

```diff
@@ -87,14 +88,48 @@ const accessLegFilter = [
 
 type Props = {
   activeLeg?: Leg;
+  colorOverride?: string;
+  showRouteArrows?: boolean;
   transitiveData?: TransitiveData;
 };
 
+const images = [
+  {
+    url: routeArrow,
+    id: "arrow-icon"
```

Comment:
Sort attributes.

Reply:
I can get this swap, sure. But I'm having a hard time getting this through the CI/CD because of the stories. I had to hoist the arrow image into the branch along with the logic to add it to the map instance on runtime. The image is causing problems with the tests because it can't find the .png file needed for it even though when it's actually running it's fine.

Do you have any experience on resolving things like this? Miles sent me something about the storybook/main.js file to poke around in, but I'd appreciate all the hand holding I can get.

Reply:
Have you tried to update the snapshots? (CI runs node 16, so try with that version if you haven't).

Reply:
I believe I tried that. That's 
```
yarn test -u

OR

yarn unit -u
```
right?

Reply:
`yarn unit -u`